### PR TITLE
Navbar changes

### LINKED ISSUE #19: Add Apply for Mentor changes + Navbar
On December 9th

### PR DESCRIPTION
**Pull request by annehu2**
#19 
- Navbar with link to scroll to section
- Rainbow border
- Link to mentor apps
- hamburger menu for mobile (currently working on media query to solve window sizing)
- vertical menu for mobile
- sticky menu

### CODE REVIEW

File: components/molecules/Layout.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { styled, css } from "~stitches";
 
-import { Footer } from "@components/molecules";
+import { Footer, Navbar } from "@components/molecules";
```
Comment (Alluq) [inline components/molecules/Layout.jsx]: why was navbar added but never used?

File: components/molecules/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  return (
+    <Wrapper id="navbar">
+      <LogoDiv>
+        <Image src={Logo} alt='official logo'></Image>
+      </LogoDiv>
+      <BrowserView>
+        <Links id="links">
+          <Link href="https://forms.gle/2AjFu2TzLVFSoDPo7">Mentor</Link>
+          {/* <Link href="">Volunteer</Link> */}
```
Comment (Alluq) [inline components/molecules/Navbar.jsx]: Remove the comments just for cleanliness

File: components/molecules/Layout.jsx (hunk 1 above)
Comment (annehu2) [reply components/molecules/Layout.jsx]: lol good point ty for catching that

File: components/molecules/Navbar.jsx (hunk 2 above)
Comment (annehu2) [reply components/molecules/Navbar.jsx]: bet

**Review by Alluq — APPROVED** [review]
LGTM

**Pull request merged**